### DISCUSSION TITLE
Updating Traefik LXC   environment: line 59: RELEASE: unbound variable

### DISCUSSION THREAD

**Discussion by davlaw** [tteck/Proxmox#3030] [Help Needed] [answered]
error when running "update" in traefik

My inexperience with treafik there might be something I missed

**Comment by tteck** [comment]
Fixed https://github.com/tteck/Proxmox/commit/2f6ed335755a760e8fc4ff11787200bee80bb560

**Comment by davlaw** [comment]
!! Fast response !!

But I get something a bit different now

 /mv: cannot stat '/tmp/traefik*/traefik': No such file or directory

[ERROR] in line 63: exit code 0: while executing command mv /tmp/traefik*/traefik /usr/bin/

**Reply by tteck** [reply]
I'll take a look, when I get a minute.

**Reply by Uzurka** [reply]
Could you provide more logs ?
The update part is supposed to : 
- Download the traefik latest release from the github releases
- extract it into /tmp
- move /tmp/traefik to /usr/bin/traefik

Actually, one of those 3 steps is failing, but without logs, can't determine which ^^

**Reply by Uzurka** [reply]
got it 
fixing rn

**Reply by Uzurka** [reply, marked as answer]
https://github.com/tteck/Proxmox/pull/3031

**Comment by davlaw** [comment]
Thanks, did not expect such quick results Kudos
